networks/suzuka/load-soak-testing/bin/basicalicebob.rs

```diff
@@ -0,0 +1,190 @@
+use anyhow::{Context, Result};
+use aptos_sdk::{
+	coin_client::CoinClient,
```

Comment:
For organization, I still want you to move these out into the appropriate network crates. `suzuka-client` for example.